lib/server.js

```diff
@@ -81,6 +81,29 @@ exports.create = function create(createOptions) {
       callback(err, html);
     }
 
+    function matching(options) {
+      var componentInstance = null;
+      match(options, function(error, redirectLocation, renderProps) {
+        if (redirectLocation) {
+          debug('server.js match 302 %s', JSON.stringify(redirectLocation));
```

Comment:
i will create another pull request for 302,404 and 500. thanks for your comments.